apis/fluent-en_GB/atrium-api-fluent-en_GB-jvm/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/PathExpectationSamples.kt

```diff
@@ -252,10 +252,10 @@ class PathExpectationSamples {
             .notToEndWith("jpg")
 
         fails {
-            expect(dir).extension //            subject is now of type String (actually "txt")
-                .toEqual("txtt") //     fails because it doesn't equal to "txtt"
-                .toEndWith("jpg") //    not reported
-            //                                  use `.extension` if you want that all expectations are evaluated
+            expect(dir).extension         // subject is now of type String (actually "txt")
```

Comment:
Could it be, that you have enabled parameter name hints in Intellij? The alignment is still not right (as you can see here)

Reply:
Think that is outdated version, this is new one
![Screenshot_45](https://user-images.githubusercontent.com/39156859/120773440-65176200-c532-11eb-9a3b-b8308a28fd24.png)

Reply:
hmm... starts to be bothersome really. Looks all good if I go to https://github.com/robstoll/atrium/pull/930/files
Doesn't look good if I come from the github notification where one should see only the new differences:
https://github.com/robstoll/atrium/pull/930/files/5dbe7c8c8db13091cbb5a22f6b669eff86f2a893..eb3113721fb5c3ba2eae5834bb73e7be378f57f1#diff-bec3c79f9b9bc972773563affa67021ede5661d57d9d9968bb81f8cc95af3780